README.md

```diff
@@ -54,13 +55,10 @@ following subprojects, which are packaged versions of DataFusion intended for en
 - [DataFusion Comet](https://github.com/apache/datafusion-comet/) is an accelerator for Apache Spark based on
   DataFusion.
 
-The target audience for the DataFusion crates in this repository are
-developers building fast and feature rich database and analytic systems,
-customized to particular workloads. See [use cases] for examples.
-
-DataFusion offers [SQL] and [`Dataframe`] APIs,
-excellent [performance], built-in support for CSV, Parquet, JSON, and Avro,
-extensive customization, and a great community.
+"Out of the box,"
+DataFusion offers [SQL] and [`Dataframe`] APIs, excellent [performance],
```

Comment:
nit: Will this render with a linebreak after `"Out of the box,"`?

Reply:
I don't believe it will. Markdown is a bit ambiguous on this and different impl's handle this differently however most either will force a newline in rendered view if the line ends with two spaces (now that is a horrible pattern) or a back slash. This comment editor is an example of something that does put text on a newline 
when a line break is encountered.